Source/Core/AudioCommon/XAudio2Stream.cpp

```diff
@@ -185,11 +231,25 @@ bool XAudio2::Start()
 
   // XAudio2 master voice
   // XAUDIO2_DEFAULT_CHANNELS instead of 2 for expansion?
-  if (FAILED(hr = m_xaudio2->CreateMasteringVoice(&m_mastering_voice, 2, m_mixer->GetSampleRate())))
+  if (SConfig::GetInstance().bDPL2Decoder)
   {
-    PanicAlert("XAudio2 master voice creation failed: %#X", hr);
-    Stop();
-    return false;
+    if (FAILED(
+            hr = m_xaudio2->CreateMasteringVoice(&m_mastering_voice, 6, m_mixer->GetSampleRate())))
```

Comment:
This code-block can be compacted with either a ternary if or by assigning the 6 (or 2), to a temporary variable based on `SConfig::GetInstance().bDPL2Decoder`.

Reply:
Done.